Athens_Univ/AU590219.txt

```diff
@@ -4,7 +4,7 @@ DATE: 2019.11.21
 AUTHORS: Dimitrios E. Damalas, Stefanos Kokolakis, Reza Aalizadeh, Nikolaos Thomaidis, University of Athens
 LICENSE: CC BY
 COPYRIGHT: Copyright (C) 2019 Department of Chemistry, University of Athens
-PUBLICATION: 
```

Comment:
Regarding publication field ... if there is no publication, please remove this field rather than writing N/A. It is not a compulsory field (but possibly still comes through incorrectly in the settings in RMassBank)

Reply:
Hi there. All the files have ION_MODE tag, maybe the order is the problem (new format). Let me check, thanks.